containers/python-3/.devcontainer/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # Copy requirements.txt (if found) to a temp locaition so we can install it. Also
 # copy "noop.txt" so the COPY instruction does not fail if no requirements.txt exists.
-COPY requirements.txt* .devcontainer/noop.txt /tmp/pip-tmp/
+COPY requirements.txt .devcontainer/noop.txt /tmp/pip-tmp/
```

Comment:
It looks like the `*` is actually needed here (at least in Docker for Windows where I tested this). What I am seeing when I test this is that it bombs on this line if no `requirements.txt` is present.  If `requirements.txt*` is used, it sill still pass if it isn't there as long as one file gets copied. Interesting behavior - not what I expected.

Reply:
Ah, I kept testing with a `requirements.txt` file since I had to make sure `pip` worked and then I forgot to take it back out. 😁

Reply:
I edited in-browser and put the `*` at the front as there's a chance people will have a `dev-requirements.txt` file and so this way makes a bit more sense. I haven't tested it yet but next time I'm in the office on my desktop and not my laptop I can give it a go.